desc/plotting.py

```diff
@@ -3613,6 +3621,227 @@ def plot_basis(basis, return_data=False, **kwargs):
         return fig, ax
 
 
+def plot_field_lines(
+    field,
+    R0,
+    Z0,
+    Phi0=0,
```

Comment:
minor point, but in our notation the toroidal angle is `phi` (lowercase), `Phi` (uppercase) usually means a potential of some kind.